demisto_sdk/commands/content_graph/content_graph_commands.py

```diff
@@ -78,9 +80,32 @@ def update_content_graph(
         content_graph_interface.clean_import_dir()
         if not imported_path:
             # getting the graph from remote, so we need to clean the import dir
-            extract_remote_import_files(content_graph_interface, builder)
-
+            try:
+                extract_remote_import_files(content_graph_interface)
+            except RuntimeError as e:
+                logger.warning(
+                    "Failed to download the content graph, will create a new graph"
+                )
+                logger.debug(f"Runtime Error: {e}")
+                create_content_graph(
+                    content_graph_interface, marketplace, dependencies, output_path
+                )
+                return
     content_graph_interface.import_graph(imported_path)
+    try:
+        # Test that the imported graph is valid by marshaling it
+        content_graph_interface.marshal_graph(MarketplaceVersions.XSOAR)
+        # we need to clear cache after validating the graph
+        content_graph_interface.clear_cache()
+    except ValidationError as e:
+        logger.warning(
+            "Failed to marshal the graph, probably the schema has changed. Will create a new graph"
+        )
+        logger.debug(f"Validation Error: {e}")
+        create_content_graph(
+            content_graph_interface, marketplace, dependencies, output_path
+        )
+        return
```

Comment:
let's move the try-except ValidationError to `import_graph` and return the success status:
```suggestion
    success = content_graph_interface.import_graph(imported_path)
    if not success:
            create_content_graph(
                content_graph_interface, marketplace, dependencies, output_path
            )
            return
```
the try-except should be called in `import_graph()` from a `is_successful_import() -> bool` method:
```
try:
    # Test that the imported graph is valid by marshaling it
    content_graph_interface.marshal_graph(MarketplaceVersions.XSOAR)
    content_graph_interface.clear_cache()
    return True
except ValidationError as e:
    logger.warning(
        "Failed to marshal the graph, probably the schema has changed. Will create a new graph"
    )
    logger.debug(f"Validation Error: {e}")
    return False
```